proxy/contracts/mainnet/MessageProxyForMainnet.sol

```diff
@@ -78,6 +75,8 @@ contract MessageProxyForMainnet is SkaleManagerClient, MessageProxy, IMessagePro
     string public version;
     bool public override messageInProgress;
 
+    mapping(bytes32 => Pause) public pauseInfo;
```

Comment:
Custom types have a big potential to improve in mappings. So we can add new variables in structure during upgrade when we would need to update this scheme, otherwise we would need to create a new mapping